src/main/java/dinkplugin/notifiers/data/DeathNotificationData.java

```diff
@@ -0,0 +1,10 @@
+package dinkplugin.notifiers.data;
+
+import lombok.Value;
+
+@Value
+public class DeathNotificationData {
+    Integer valueLost;
```

Comment:
```suggestion
    Long valueLost;
```

overflow concern also carries through here

Reply:
Updated in [609fcd8](https://github.com/pajlads/DinkPlugin/pull/55/commits/609fcd817ad66494ed57c5f6ffa2df2298cc9441)